scripts/upgrade

```diff
@@ -143,6 +143,10 @@ then
     # Fix warning "Your installation has no default phone region set"
     if [ "$(exec_occ config:system:get default_phone_region)" == "" ]; then
 	config_locale=$(exec_occ config:system:get default_locale)
+	if [ "$config_locale" == "" ]; then
+		# If it's not set in config use "US" as default
+		config_locale="US"
```

Comment:
Why not FR by default?

Reply:
Why not any other code. It would be more logical that the user chooses from a drop down list at the installation or change it in the app panel

Reply:
We are talking about a default value set if there was no previous value set.

Reply:
Hi @ericgaspar,
I set "US" because in `manifest.toml` I used "en_US" as default install language and so I wanted to use the default country here as well.

Reply:
let's then change the default language to `fr` (because it is by far the most used...)